chore(gen): docs content review 2024-08-26 (#3626)
scaleway/docs-content: compute/instances/how-to/use-boot-modes.mdx

---
meta:
  title: How to use boot modes on Instances
  description: This page explains how to use boot modes for Scaleway Instances.
content:
  h1: How to use boot modes on Instances
  paragraph: This page explains how to use boot modes for Scaleway Instances.
tags: boot-mode boot mode boot instance bootscript
dates:
  validation: 2024-08-26
  posted: 2021-05-26
categories:
  - compute
---

You can boot your Instances using local boot or a resuce image. This page explains how to use each of these boot modes.

<Message type="tip">
  - You can choose to use **cloud-init** when [creating an Instance](/compute/instances/how-to/create-an-instance/) via **Advanced settings**. This enables you to control the mode of the Instance's very first boot.

>   - Your Instance must be [powered off](/compute/instances/how-to/power-off-instance/) in order to change the boot mode of it.

Reviewer: ```suggestion
  - Your Instance must be [powered off](/compute/instances/how-to/power-off-instance/) in order to change its boot mode.
```
